clang/include/clang/Serialization/ASTBitCodes.h

```diff
@@ -70,38 +71,53 @@ using DeclID = DeclIDBase::DeclID;
 
 /// An ID number that refers to a type in an AST file.
 ///
-/// The ID of a type is partitioned into two parts: the lower
+/// The ID of a type is partitioned into three parts:
+/// - the lower
 /// three bits are used to store the const/volatile/restrict
-/// qualifiers (as with QualType) and the upper bits provide a
-/// type index. The type index values are partitioned into two
+/// qualifiers (as with QualType).
+/// - the upper 29 bits provide a type index in the corresponding
```

Comment:
```suggestion
/// - the next 29 bits provide a type index in the corresponding
````
NIT: maybe update the wording? Upon the first read I thought it's referring to the 29 highest bit and only after reading the next bullet item I have realized it's referring to the 29 bits after the initial 3 lower bits.

Reply:
Done